### PR TITLE
ENCD-3669 Rename calculatedProperty to notSubmittable to more accurately describe its meaning

### PR DESCRIPTION
**Pull request by davisagli**
(Needed for ENCD-3669)

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -6,6 +6,7 @@ extends = versions.cfg
 allow-hosts =
     *.python.org
     github.com
+index = https://pypi.python.org/simple/
```
Comment (caseylitton) [inline buildout.cfg]: remove and rebase once this change is merged to master from ENCD-3684

File: CHANGES.rst [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 - add port_range to wsgi_tests (mrmin)
 
+- Use notSubmittable instead of calculatedProperty to indicate properties that may not be submitted.
```
Comment (caseylitton) [inline CHANGES.rst]: The last release was version 0.30.  I assume this should be on '0.31' but we don't have any release notes between .24 and .30.  I guess we should add version 0.31 to the changes in the buildout.cfg PR. Then tack this PR change below it in here.

Comment (davisagli) [reply CHANGES.rst]: Where are you seeing that 0.30 release? It's not on PyPI: https://pypi.python.org/simple/snovault/

**Comment by hitz** [comment]
I was lazy and did not submit 0.30 to pypi.

File: CHANGES.rst (hunk 2 above)
Comment (davisagli) [reply CHANGES.rst]: I do see a 0.30 tag, but not a release on PyPI. @hitz @Bek did someone make a tag without updating the changelog or publishing it to PyPI?

**Comment by davisagli** [comment]
I rebased.

**Pull request merged**